### PR TITLE
Documentation Naming Convention Todos Tutorial

### PR DESCRIPTION
**Pull request by lsaudon**
## Status
**READY**

## Breaking Changes
NO

## Description
Todos Tutorial follow the naming convention.

## Todos
- [x] Tests
- [x] Documentation
- [x] Examples

### CODE REVIEW

**Comment by lsaudon** [comment]
I don't know what to put in place of **ToggleAll** Event.

**Comment by lsaudon** [comment]
#934

**Comment by lsaudon** [comment]
I hope I didn't make a mistake.

**Comment by lsaudon** [comment]
It's good for me.

File: docs/cs/fluttertodostutorial.md [hunk 1]
```diff
@@ -320,7 +320,7 @@ class TodosBloc extends Bloc<TodosEvent, TodosState> {
   TodosBloc({@required this.todosRepository});
 
   @override
-  TodosState get initialState => TodosLoading();
+  TodosState get initialState => TodosLoadInProgress();
 
   @override
   Stream<TodosState> mapEventToState(TodosEvent event) async* {
```
Comment (felangel) [inline docs/cs/fluttertodostutorial.md]: I think we need to also update the `mapEventToState`

```dart
if (event is TodosLoaded) {
  yield* _mapTodosLoadedToState();
} else if (...) {...}
```

File: examples/flutter_todos/lib/widgets/stats.dart [hunk 2]
```diff
@@ -14,9 +14,10 @@ class Stats extends StatelessWidget {
   Widget build(BuildContext context) {
     return BlocBuilder<StatsBloc, StatsState>(
       builder: (context, state) {
-        if (state is StatsLoading) {
-          return LoadingIndicator(key: FlutterTodosKeys.statsLoadingIndicator);
-        } else if (state is StatsLoaded) {
+        if (state is StatsLoadInProgress) {
+          return LoadingIndicator(
+              key: FlutterTodosKeys.statsLoadInProgressIndicator);
```
Comment (felangel) [inline examples/flutter_todos/lib/widgets/stats.dart]:
```suggestion
              key: FlutterTodosKeys.statsLoadInProgressIndicator,
          );
```

File: docs/cs/fluttertodostutorial.md (hunk 1 above)
Comment (felangel) [inline docs/cs/fluttertodostutorial.md]: don't we have to update the actual body of `mapEventToState` to use the updated names?

File: examples/flutter_todos/lib/blocs/todos/todos_bloc.dart [hunk 3]
```diff
@@ -16,21 +16,21 @@ class TodosBloc extends Bloc<TodosEvent, TodosState> {
   @override
   Stream<TodosState> mapEventToState(TodosEvent event) async* {
     if (event is TodosLoaded) {
-      yield* _mapLoadTodosToState();
+      yield* _mapTodosLoadedToState();
```
Comment (felangel) [inline examples/flutter_todos/lib/blocs/todos/todos_bloc.dart]: I think we need to apply these changes to the docs as well

File: docs/cs/fluttertodostutorial.md (hunk 1 above)
Comment (lsaudon) [reply docs/cs/fluttertodostutorial.md]: sorry i just change tutorial not doc

**Comment by lsaudon** [comment]
Sorry for the waste of time.

**Comment by lsaudon** [comment]
All good?

**Pull request merged**